native/SampleApps/RestAPIExplorer/RestAPIExplorer/Classes/AppDelegate.m

```diff
@@ -48,9 +48,6 @@ - (id)init
     self = [super init];
     if (self) {
         [SFAuthenticationManager sharedManager].advancedAuthConfiguration = SFOAuthAdvancedAuthConfigurationRequire;
-        [SalesforceSDKManager sharedManager].connectedAppId = RemoteAccessConsumerKey;
```

Comment:
Example code for RestAPIExplorer.  Explicit configuration in code no longer necessary.